src/sidebar/index.js

```diff
@@ -1,11 +1,41 @@
-import { parseJsonConfig } from '../boot/parse-json-config';
+// Load polyfill for :focus-visible pseudo-class.
+import 'focus-visible';
+// The entry point component for the app.
```

Comment:
The "The entry point component" comment is now out of place. I suggest we just remove it.